scalr/README.md

```diff
@@ -0,0 +1,71 @@
+# Scalr Integration
+
+## Overview
+
+This integration collects metrics from Scalr.
```

Comment:
Could you provide more insight on what this integration does and the value of the integration?

Reply:
For example: 

1. What does scalr do in 1-2 sentences?
2. What will customers get from this integration 
3. What types of metrics?